tools/compute_font_metrics.py

```diff
@@ -98,6 +98,23 @@
   "Redacted"
 ]
 
+
+def generate_italic_angle_images():
+  from PIL import Image, ImageDraw
+  import math
+  for i in range(10):
+    angle = 30*(float(i)/10) * 3.1415/180
+    im = Image.new('RGBA', (200,50), (255,255,255,0))
+    N = 15
+    spacing = 200/N
```

Comment:
I'll make a PR for this now and a few other things, I got the images working much better